inc/classes/busting/class-google-analytics.php

```diff
@@ -40,7 +40,7 @@ class Google_Analytics extends Abstract_Busting {
 	 * @access protected
 	 * @author Grégory Viguier
 	 */
-	protected $filename = 'ga-%s.js';
+	protected $file_name_pattern = 'ga-%s.js';
```

Comment:
As we touched this file, let's move it into the module and rename.

Reply:
Done.